src/interpreter.jl

```diff
@@ -38,6 +38,12 @@ function interpret(tab::SymbolTable, ex::Expr)
         else
             return tab[args[1]](interpret.(Ref(tab),args[2:end])...)
         end
+    elseif ex.head == :(.)
+        return Base.materialize(Base.broadcasted(Base.eval(args[1]), interpret(tab, args[2])...))

```

Comment:
Not sure if this is the best approach to do so for broadcasting.

Reply:
It seems from the CI that `Base. materialize` was not defined in Julia 1.0. I will look into what's the universal way to achieve so for all Julia >=1.0

Reply:
Sounds good. Thanks!

Reply:
Changing `Base.materialize(Base.broadcasted(...))` to `Base.broadcast(...)` as in https://github.com/sisl/ExprRules.jl/pull/25/commits/8c341fdaab87e096b245f583249132c6b628adff fixes the issue :)

Reply:
Yay! I'll talk with @rcnlee today and get this merged! Thanks!

Reply:
Thanks @xukai92, this pull request looks good to me!  I'll merge it.